### ISSUE TITLE
Embedded mp3 player issues

### ISSUE THREAD

**Issue by coregis** [eldang/ballard-4culture#22]
- [ ] It is not possible to move forward or backward in time using the time slider in the embedded mp3 player
- [x] If an audio file is playing in a pop-up, and the pop-up is closed using the closed button, the player persists and the audio continues playing until another pop-up is generated
- [x] It is possible to play all of the mp3 files in a pop-up simultaneously; I'm not sure what kind of maniac would do this so it's probably not a big deal
![image](https://user-images.githubusercontent.com/8813214/149435078-5ad6e64d-da19-4ccf-9c11-803b05a8e590.png)

**Comment by coregis** [comment]
@eldang if it's going to be involved (like more than an hour or so of your time) to address these I'm fine to leave these behaviors as-is

**Comment by eldang** [comment]
omg I'm testing this and it is most certainly possible to make it sound like one is losing one's mind.

**Comment by eldang** [comment]
@coregis I am finding that the time slider works, within the constraint that I can only go as far as the file has been loaded so far.  Can you take a second look and see if that's the issue you're having or if it's a more complete failure?

Meanwhile I'm focussing on the middle point which I think is the one most likely to really frustrate users.  I notice that it also applies to switching tabs within a popup, which is possibly worse.  I'm going to see if there's a way to write a function that stops all audio, and have it triggered by:

1. The close popup button, which would resolve the specific issue you'd noticed.
2. Clicking on a tab header, which would resolve the version I just noticed.
3. Possibly starting any other audio, which would resolve your last observation.  Though like you I'm much less concerned about that one.  If I can get the other parts of this working, then there won't be a situation of multiple clips playing that isn't easy for the user to sort out for theirself.

**Comment by coregis** [comment]
@eldang Ha ha yes it definitely has a crazy-making capability! I am experiencing the time-slider working within the limit of the loaded file in Firefox, but in Chrome (at least on my Windows 10 laptop) I have no ability to move within the audio at all. 

I think your proposed solutions for the middle point are great. And honestly I'm fine if we can't get the time slider in the mp3 player to work in Chrome.

**Comment by eldang** [comment]
You're right about the time slider - I also can't get it to work in Chrome.  But I think that probably isn't worth digging into now.  #23 resolves the other issues, which turned out to be a little simpler to do than I had expected.
